modules/cloudflare_form_tester/src/Mocks/ComposerDependenciesCheckMock.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\cloudflare\Mocks\ComposerDependenciesCheckMock.
+ */
+
+namespace Drupal\cloudflare_form_tester\Mocks;
+
+use Psr\Log\LoggerInterface;
+use Drupal\cloudflare\Exception\ComposerDependencyException;
+use Drupal\cloudflare\CloudFlareComposerDependenciesCheckInterface;
+
+/**
+ * Tests that composer dependencies are met.
+ */
+class ComposerDependenciesCheckMock implements CloudFlareComposerDependenciesCheckInterface {
+
+  /**
+   * Set the first time that this function is called in a bootstrap.
+   *
+   * @var bool
+   */
+  public static $areDependenciesMet = NULL;
+
+  /**
+   * A logger instance.
+   *
+   * @var \Psr\Log\LoggerInterface
+   */
+  protected $logger;
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(LoggerInterface $logger) {
```

Comment:
This should have the same prototype as ComposerDependenciesCheck so that it can be swapped out.